src/types.ts

```diff
@@ -620,6 +622,64 @@ export enum AccountTransactionType {
     TransferWithScheduleWithMemo = 24,
 }
 
+export enum ModuleTransactionType {
```

Comment:
This appears to be unused, and so does `export interface ModuleTransaction` below. Please remove it.